spec/lib/fields/time_spec.rb

```diff
@@ -13,4 +13,30 @@
       expect(path).to eq("/fields/time/#{page}")
     end
   end
+
+  describe "#time" do
+    before do
+      @time = DateTime.new(2021, 3, 26, 16, 38)
+    end
```

Comment:
And `before`! I'd just put the `DateTime` definition in each `it` block.

Reply:
Sorry, I totally missed the second part of your original comment! 🤦‍♂️